Wrapping/Generators/Python/module_ext.i.in

```diff
@@ -4,13 +4,8 @@
 // Needs to be investigated
 #include <iostream>
 
-#if PY_VERSION_HEX >= 0x03020000
-# define SWIGPY_SLICE_ARG(obj) ((PyObject*) (obj))
-# define SWIGPY_SLICEOBJECT PyObject
-#else
-# define SWIGPY_SLICE_ARG(obj) ((PySliceObject*) (obj))
-# define SWIGPY_SLICEOBJECT PySliceObject
-#endif
+#define SWIGPY_SLICE_ARG(obj) ((PyObject*) (obj))
```

Comment:
Thanks Matt! If I understand correctly, we may just remove `SWIGPY_SLICE_ARG` as well, once we have upgraded SWIG. `SWIGPY_SLICE_ARG` appears to be replaced with `SWIGPY_SLICEOBJECT`, by pull request https://github.com/swig/swig/pull/2344 commit https://github.com/swig/swig/commit/e683168018558f4e73dc26629c4bbec224d23438 by @clintonstimpson, September 2022 (included with the latest SWIG release, version 4.2.1).

Reply:
No, when I tested it was still needed.

Reply:
https://github.com/swig/swig/commit/e683168018558f4e73dc26629c4bbec224d23438 (September 2022) has removed the `SWIGPY_SLICE_ARG(obj)` macro, so why would you still need it, then?

Reply:
By the way, SWIG already appears to require Python >= 3.3, when using Python 3. But it seems to me that SWIG still support Python 2 as well! Could that be the reason why they still have `SWIGPY_SLICEOBJECT`? Otherwise they might as well drop `SWIGPY_SLICEOBJECT`. 🤔 
https://github.com/swig/swig/blob/a7eca8b4159840801c911c0d7b2a3325c3cbfa1e/Lib/python/pyrun.swg#L10-L16

Reply:
Thanks Matt. When SWIG is upgraded, I think we can try to see if the CI is OK with dropping `SWIGPY_SLICE_ARG(obj)`.

The other macro, `SWIGPY_SLICEOBJECT`, may still be needed for a while, though. At least, it's still in use by the current SWIG revision.